src/richie/apps/courses/models/course.py

```diff
@@ -198,6 +198,13 @@ def __str__(self):
             title=self.extended_object.get_title(),
         )
 
+    @property
+    def pt_effort(self):
+        """Return effort as a PT string for schema.org metadata."""
```

Comment:
What is exactly a PT string? Is it something related to open graph?

Reply:
Apparently it is part of ISO8601: https://fr.wikipedia.org/wiki/ISO_8601#Durée

Reply:
Oh ok !!